topics/assembly/tutorials/vgp_genome_assembly/tutorial.md

```diff
@@ -342,32 +350,32 @@ In order to identify some key characteristics of the genome, we do genome profil
 
 > <hands-on-title>Generate <i>k</i>-mers count distribution</hands-on-title>
 >
->**Step 1**: Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} with the following parameters:
->  1. *"Operation type selector"*: `Count operations`
->  2. *"Count operations"*: `Count: count the occurrences of canonical k-mers`
->  3. {% icon param-collection %} *"Input sequences"*: `HiFi_collection (trim)`
->  4. *"k-mer size selector"*: `Set a k-mer size`
->  5. "*k-mer size*": `31`
+> 1. Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} with the following parameters:
+>    - *"Operation type selector"*: `Count operations`
+>    - *"Count operations"*: `Count: count the occurrences of canonical k-mers`
+>    - {% icon param-collection %} *"Input sequences"*: `HiFi_collection (trim)`
+>    - *"k-mer size selector"*: `Set a k-mer size`
+>    - "*k-mer size*": `31`
 >
-> > <comment-title>Selection of <i>k</i>-mer size</comment-title>
-> >
-> > We used 31 as *k*-mer size, as this length has demonstrated to be sufficiently long that most *k*-mers are not repetitive and is short enough to be more robust to sequencing errors. For very large (haploid size > 10 Gb) and/or very repetitive genomes, larger *k*-mer length is recommended to increase the number of unique *k*-mers.
-> {: .comment}
-><br>
->**Step 2**: Rename output as `meryldb`
+>    > <comment-title>Selection of <i>k</i>-mer size</comment-title>
```

Comment:
do we really need a space between the two `>`?

Reply:
Without the space, the list numbering restarts after the comment box.